src/test/java/com/google/api/generator/engine/ast/ConcreteReferenceTest.java

```diff
@@ -30,6 +30,30 @@
   public void basicConcreteReference() {
     Reference reference = ConcreteReference.builder().setClazz(Integer.class).build();
     assertEquals(reference.name(), Integer.class.getSimpleName());
+    assertFalse(reference.isStaticImport());
+  }
+
+  @Test
+  public void basicConcreteReference_setIsStaticImport() {
+    Reference reference =
+        ConcreteReference.builder().setClazz(Integer.class).setIsStaticImport(true).build();
+    assertEquals(reference.name(), Integer.class.getSimpleName());
+    assertFalse(reference.isStaticImport());
```

Comment:
Sorry, I confused the value of `isStaticImport`, why should be `false` since `setIsStaticImport(true)` at line 39